CREDITS.md

```diff
@@ -76,11 +78,13 @@ Contributors to the Firecracker repository:
 - Christopher Diehl <diehl.chris24@gmail.com>
 - Christos Katsakioris <ckatsak@cslab.ece.ntua.gr>
 - cneira <cneirabustos@gmail.com>
+- Colin Percival <cperciva@freebsd.org>
```

Comment:
this one could get merged in `.mailmap`?

Reply:
good catch! will fix it soon!

Reply:
fixed.